### PR TITLE
DDPB-1147-align-settings

### PR DESCRIPTION
**Pull request by jackgoodby**
Co-dependency with API PR of same name

### CODE REVIEW

File: app/config/security.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ security:
         # /admin accessible only from ADMIN, super users and AD users
         - { path: ^/admin, roles: [ROLE_ADMIN, ROLE_AD] }
         # client settings not available to PA users
-        - { path: ^/settings, roles: [ROLE_DEPUTY, ROLE_ADMIN, ROLE_AD] }
```
Comment (slizzio) [inline app/config/security.yml]: Q: Did you mean to remove this line?

Comment (slizzio) [inline app/config/security.yml]: Forget this, noticed the controller has been removed :)

File: src/AppBundle/Form/Settings/ProfileType.php [hunk 2]
```diff
@@ -14,29 +14,38 @@
...
-    public function __construct(User $loggedInUser)
+    public function __construct($validationGroups)
     {
-        $this->loggedInUser = $loggedInUser;
+        $this->validationGroups = $validationGroups;
     }
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $loggedInUser = $builder->getData();
```
Comment (slizzio) [inline src/AppBundle/Form/Settings/ProfileType.php]: Q: Not sure why this is different from the loggedInUser that was injected into the class?

File: src/AppBundle/Controller/SettingsController.php [hunk 3]
```diff
@@ -2,13 +2,18 @@
...
 
+use Symfony\Component\HttpFoundation\Request;
+
 class SettingsController extends AbstractController
 {
     /**
-     * @Route("/settings", name="account_settings")
+     * @Route("/deputyship-details", name="account_settings")
+     * @Route("/pa/settings", name="pa_settings")
      * @Template()
```
Comment (elvisciotti) [inline src/AppBundle/Controller/SettingsController.php]: two routes. worth merging ?  I'll keep reading the PR and understad more

File: src/AppBundle/Controller/SettingsController.php [hunk 4]
```diff
@@ -20,4 +25,115 @@ public function indexAction()
...
         ];
     }
-}
+
+    /**
+     * @Route("/deputyship-details/your-details/change-password", name="user_password_edit")
+     * @Route("/pa/settings/your-details/change-password", name="pa_profile_password_edit")
+     * @Template()
+     */
+    public function passwordEditAction(Request $request)
```
Comment (elvisciotti) [inline src/AppBundle/Controller/SettingsController.php]: git doesn't show the differences with the old action in the deleted controller, so difficult to understand what changed here

**Comment by elvisciotti** [comment]
ticked is 1447. not 1147

**Pull request merged**